wiki/Performance_Troubleshooting/vi.md

```diff
@@ -0,0 +1,106 @@
+- Nếu bạn sử dụng driver không chính thức, hãy thử đổi sang bản chính thức từ nhà phát hành.
+- Hãy đảm bảo bạn chạy osu! ở chế độ Toàn màn hình. Chạy ở chế độ Cửa sổ hay Không viền làm tăng độ trễ khoảng một khung hình.
+
+### Mất điều khiển
+
+Khi đang chơi, một hoặc nhiều thiết bị đầu vào ngừng phản hồi. Mọi thứ trên màn hình vẫn di chuyển đúng như thường, nhưng bạn không thể chơi cho đến khi bạn có thể điều khiển lại. Khi chẩn đoán hoặc báo cáo, bạn hãy thử:
+
+- Bấm Ctrl+F11 để xem biểu đồ biểu thị thời gian khung hình. Kiểm tra xem liệu có xuất hiện nhiều đốm/vạch trắng trên biểu đồ khi việc này xảy ra hay không (nó có nghĩa là một ứng dụng khác, hoặc driver trên hệ thống của bạn đang dùng quá nhiều CPU time, làm trễ tín hiệu vào).
+
+### FPS thấp
+
+Khi chơi, FPS thấp quá làm cho các vật thể xuất hiện trên màn hình trông giật và di chuyển kỳ quặc. Điều này cũng có thể khiến tăng độ trễ của tín hiệu vào. Khi chẩn đoán hoặc báo cáo, bạn hãy thử:
+
+- Kiểm tra xem liệu nó vẫn xảy ra sau khi khởi động lại máy.
+- - Thử đóng các ứng dụng mà hiện overlay trên game (như Discord, Steam, NVIDIA Shadowplay, v.v.)
+- Đảm bảo bạn không bật đồng bộ dọc (Vertical Sync) ở bất cứ driver của card đồ hoạ nào (nên để tắt "Off" hoặc tuỳ thiết lập của ứng dụng "Application-controlled")
+- Thử thay đổi Giới hạn khung hình xem liệu nó có cải thiện tình trạng hay không.
+- Hãy đảm bảo bạn chạy osu! ở chế độ Toàn màn hình (không phải Không viền). Điều này sẽ luôn cải thiện hiệu năng.
+- Đảm bảo bạn đang bật [Game mode](https://quantrimang.com/huong-dan-kich-hoat-che-do-game-mode-tren-windows-10-130041). Nó cho phép Windows cung cấp nhân (CPU/GPU) và tài nguyên dành riêng cho osu!.
+- Mở Task manager, chuyển sang tab Details và sắp xếp danh sách theo thứ tự tiêu tốn CPU. Kiểm tra xem mỗi khi xảy ra đứng khung hình, có ứng dụng nào tiêu tốn CPU nhiều không, như virus chạy dưới nền, hoặc trình quét diệt virus. Bạn có thể chạy osu! ở chế độ cửa sổ, hoặc để Task Manager ở màn hình thứ hai để chẩn đoán hiện tượng khi đang chơi hoặc chạy beatmap với mod Tự động.
+
+## Giới hạn khung hình
+
+osu! cho bạn nhiều lựa chọn giới hạn khung hình. Chúng đều có ưu và nhược điểm riêng. Chú ý rằng ở bất cứ thiết đặt nào, osu! sẽ giới hạn khung hình ở menu để hạn chế việc sử dụng CPU.
+
+### VSync (Đồng bộ dọc)
+
+Mức giới hạn này phụ thuộc vào cơ chế đồng bộ dọc của driver. Nó đảm bảo không có hiện tượng xé hình, nhưng sẽ làm tăng độ trễ khoảng 1-2 khung hình vì cần phải buffer khung hình trước khi hiển thị.
+
+Không khuyến nghị cho màn hình 60Hz. Nhưng nó có thể hữu ích hơn ở màn hình trên 120Hz, nhưng với màn hình có GSync hay FreeSync, thì việc dùng VSync để tránh xé hình không còn cần thiết nữa.
+
+### Power Saving (Tiết kiệm điện)
+
+Mức giới hạn này nhắm tới việc tiết kiệm điện trong khi vẫn đáp ứng được trải nghiệm game ổn định. Game sẽ chạy với fps bằng 2 lần tần số quét của màn hình và cố gắng giữ ổn định tần suất khung hình.
```

Comment:
```suggestion
Mức giới hạn này nhắm tới việc tiết kiệm điện trong khi vẫn đáp ứng được trải nghiệm game ổn định. Game sẽ chạy với tần suất khung hình bằng 2 lần tần số quét của màn hình và cố gắng giữ ổn định tần suất khung hình.
```

Reply:
Same reason above, but I will capitalize it to `FPS`.
Many ppl know that `fps` means `frame per seconds`. Although there's `per seconds`, few ppl know that it's `frequency` or `tần suất`.
VN osu! playerbase are mostly under-college to begin with, they usually know that the higher the fps the better from other games, but not likely knowing that **a stable fps** would be better.